agent/taskresource/volume/dockervolume.go

```diff
@@ -146,12 +255,17 @@ func (vol *VolumeResource) GetMountPoint() string {
 	return vol.VolumeConfig.Mountpoint
 }
 
+// SourcePath is fulfilling the HostVolume interface
+func (vol *VolumeResource) SourcePath() string {
+	return vol.GetMountPoint()
+}
+
 // Create performs resource creation
 func (vol *VolumeResource) Create() error {
 	seelog.Debugf("Creating volume with name %s using driver %s", vol.Name, vol.VolumeConfig.Driver)
```

Comment:
nit: should this be `vol.DockerVolumeName`?

Reply:
Yes, I will fix this in the other PR.